docs/asciidoc/modules/ROOT/partials/usage/apoc.meta.data.adoc

```diff
@@ -36,13 +59,57 @@ CALL apoc.meta.data();
+| "Something"  | "foo"        | 0     | false  | false | false     | "STRING"       | false | null   | 0    | 0     | []                             | []          | "node"
+| "Else"       | "RELATED_TO" | 2     | false  | false | false     | "RELATIONSHIP" | false | null   | 1    | 0     | ["Movie"]                      | []          | "node"
+| "Else"       | "foo"        | 0     | false  | false | false     | "STRING"       | false | null   | 0    | 0     | []                             | []          | "node"
+| "Another"    | "RELATED_TO" | 1     | false  | false | false     | "RELATIONSHIP" | true  | null   | 3    | 0     | ["Movie"]                      | []          | "node"
+| "Another"    | "ACTED_IN"   | 1     | false  | false | false     | "RELATIONSHIP" | true  | null   | 5    | 0     | ["Movie"]                      | []          | "node"
+| "Another"    | "LIKES"      | 1     | false  | false | false     | "RELATIONSHIP" | true  | null   | 7    | 1     | ["Movie", "Person"]            | []          | "node"
+| "Another"    | "born"       | 0     | false  | false | false     | "INTEGER"      | false | null   | 0    | 0     | []                             | []          | "node"
+| "Another"    | "name"       | 0     | false  | false | false     | "STRING"       | false | null   | 0    | 0     | []                             | []          | "node"
+|===
+
+The `unique` column shows if there is an unique constraint in that specific label and property.
+Similarly, the `index` column show if there is an index or not, while the `existence` looks for an existence constraint.
+
+The `array` column check if the row is of type array and, in case `type` columns is `"RELATIONSHIP"`,
+the result will be `true` if there is at least one node with 2 outgoing relationships with the type of relation given by `label` or `property` column, 
+so `ACTED_IN` in the example above.
+
+The `left`, `right` and `count` columns regard only rows with column `type` equals to `"RELATIONSHIP"` (otherwise they are equal to `0`).
+Please note that, because we examine a sample, these counts are just estimates to give an overview and proximity to actual values depends on the dataset and the sample set (default `100`).
+
+In particular the `count` column indicates the number of nodes with an outgoing relationship 
+(e.g. the row with label = `ACTED_IN` and property = `Person` has `count` 2 because there are 2 nodes `(node:Person)-[:ACTED_IN]->()`, i.e. `(Keanu)` and `(TomH)`).
+
+The `left` value represents the ratio (rounded down) of the sum of outgoing node-degrees to `count`, for each relationship.
+For example, regarding the row with label = `RELATED_TO` and property = `Movie`, has an outgoing node-degree equals to `1` ,because there is one relationship `(:Movie)-[rel:RELATED_TO]->()`, i.e. `(TheMatrix)-[:RELATED_TO {rate:34}]->(Laurence)` and `(TheMatrixReloaded)-[:RELATED_TO {rate:345}]->(LanaW)`.
```

Comment:
```suggestion
For example, the row with label = `RELATED_TO` and property = `Movie` has an outgoing node-degree equal to `1`, because there is one relationship `(:Movie)-[rel:RELATED_TO]->()`, i.e. `(TheMatrix)-[:RELATED_TO {rate:34}]->(Laurence)` and `(TheMatrixReloaded)-[:RELATED_TO {rate:345}]->(LanaW)`.
```